### DISCUSSION TITLE
VSCode tailwind intellisense for CSS in JS

### DISCUSSION THREAD

**Discussion by srid** [tw-in-js/twind#166] [Ideas]
There is already tailwind intellisense plugin in VSCode to highlight and autocomplete tailwind classes in HTML tags. And it would be great if something like that existed for the script tags where twind.setup defines preflight tag styling. As you can see from thie screenshot, intellisense is missing for the preflight styles.

![image](https://user-images.githubusercontent.com/3998/112935436-c44f9500-90f1-11eb-909a-b0dbee17f8f1.png)

**Comment by danielweck** [comment]
Have you tried using https://github.com/tw-in-js/vscode-twind-intellisense/
instead of https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss ?

Don't forget to try this in your VSCode settings too:
```
  "editor.quickSuggestions": {
    "strings": true
  }
```

**Reply by srid** [reply]
On a new computer I used just the twind extension, and added that setting. Has no effect, and I don't see highlighting/completion on HTML either.

You can use this file as a test case: https://github.com/srid/Feather/blob/0f7109f40a79b10a67f4d2fa67c5825ff4b52e0d/example/templates/_layout.liquid

**Reply by sastan** [reply]
Only js, jsx, ts and tsx are currently supported.

HTML, Svelte & Vue are not yet supported. I'm working on support for those.

https://github.com/tw-in-js/vscode-twind-intellisense/issues/1
